Controller/HelperController.php

```diff
@@ -444,7 +443,7 @@ public function retrieveAutocompleteItemsAction(Request $request)
      * @param AdminInterface $admin
      * @param string         $field
      *
-     * @return FormInterface
+     * @return \Sonata\AdminBundle\Admin\FieldDescriptionInterface
```

Comment:
Ok, but the main change was about `FormInterface` => `FieldDescriptionInterface`